docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>nit: <code># Imports.</code>  (or remove, it's kindof obvious)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Removed. </p>